fix: move openapi into sub dir (#40)
tensorchord/modelz-py: README.md

--framework mosec \
--name stable-diffusion-mosec
```
The result might be something like:
```json
{
  "spec": {
    "deployment_resource": {},
    "deployment_source": {
      "docker": {
        "image": "us-central1-docker.pkg.dev/nth-guide-378813/modelzai/mosec-stable-diffusion:23.04.1"
      }
    },
    "framework": "mosec",
    "http_probe_path": "/",
    "id": "0a93636b-5ed3-4abd-8fac-8a7c5a4026c9",
    "image_config": {
      "enable_cache_optimize": false
    },
    "max_replicas": 1,
    "min_replicas": 0,
    "name": "stable-diffusion-mosec",
    "server_resource": "nvidia-tesla-t4-4c-16g",
    "startup_duration": 300,
    "target_load": 10,
    "zero_duration": 300
  },
  "status": {
    "available_replicas": 0,
    "innocation_count": 0,
    "replicas": 0
  }
}
```

#### Step 2: Get Inference Endpoint

> After a some time, you could get endpoint of deployment from `list` command.

Reviewer: ```suggestion
After a while, you could get the deployment endpoint from `list` command.
```
